app/controllers/sites_controller.rb

```diff
@@ -2,15 +2,23 @@ class SitesController < ApplicationController
   before_action :user_required
 
   def index
-    @sites = Site.order(created_at: :desc).page params[:page]
+    @sites = Site.with_attached_image.includes(:user).order(created_at: :desc).page params[:page]
```

Comment:
Saw a bunch of N+1 queries in the logs when rendering sites index. 

This change fixes this (`.with_attached_image` is a nifty helper from ActiveStorage that basically adds an `.includes(attachment: :blob)`)